.github/workflows/ci-model-regression-on-schedule.yml

```diff
@@ -340,63 +368,89 @@ jobs:
 
       - name: Analyse Performance 🔍
         id: performance
+        env:
+          THRESHOLD: -0.05
```

Comment:
I think we can move it at the top to the `env:` and change the name to something like `PERFORMANCE_DROP_THRESHOLD`

Reply:
On it!